hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java

```diff
@@ -142,18 +142,18 @@ private void createPrincipals() throws Exception {
     String host = InetAddress.getLocalHost()
         .getCanonicalHostName().toLowerCase();
     String hostAndRealm = host + "@" + this.realm;
-    this.adminPrincipal = adminUser + "/" + hostAndRealm;
-    this.adminPrincipalInOtherHost = adminUser + "/otherhost@" + this.realm;
-    this.adminKeytab = new File(workDir, adminUser + ".keytab");
-    this.adminKeytabInOtherHost = new File(workDir, adminUser +
+    this.adminPrincipal = ADMIN_USER + "/" + hostAndRealm;
+    this.adminPrincipalInOtherHost = ADMIN_USER + "/otherhost@" + this.realm;
+    this.adminKeytab = new File(workDir, ADMIN_USER + ".keytab");
+    this.adminKeytabInOtherHost = new File(workDir, ADMIN_USER +
         "InOtherHost.keytab");
     createPrincipal(this.adminKeytab, adminPrincipal);
     createPrincipal(this.adminKeytabInOtherHost, adminPrincipalInOtherHost);
 
-    this.userPrincipal1 = this.user1 + "/" + hostAndRealm;
-    this.userPrincipal2 = this.user2 + "/" + hostAndRealm;
-    this.userKeytab1  = new File(workDir, this.user1 + ".keytab");
-    this.userKeytab2  = new File(workDir, this.user2 + ".keytab");
+    this.userPrincipal1 = this.USER_1 + "/" + hostAndRealm;
+    this.userPrincipal2 = this.USER_2 + "/" + hostAndRealm;
+    this.userKeytab1  = new File(workDir, this.USER_1 + ".keytab");
+    this.userKeytab2  = new File(workDir, this.USER_2 + ".keytab");
```

Comment:
Better not access `static` via instance (`this`):

```suggestion
    this.userPrincipal1 = USER_1 + "/" + hostAndRealm;
    this.userPrincipal2 = USER_2 + "/" + hostAndRealm;
    this.userKeytab1  = new File(workDir, USER_1 + ".keytab");
    this.userKeytab2  = new File(workDir, USER_2 + ".keytab");
```